app/templates/editorconfig-spaces

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+indent_style = space
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.yml]
+indent_style = space
+indent_size = 2
```

Comment:
You also need to prompt whether it should be 2 or 4 spaces. 4 spaces if actually quite common in the JS world.